hawkular-alerts-rest-tests/pom.xml

```diff
@@ -147,6 +165,19 @@
               </execution>
             </executions>
           </plugin>
+        </plugins>
+      </build>
+    </profile>
+    <profile>
+      <id>wildfly</id>
```

Comment:
Why not let the Wildfly plugin run by default when `rest` profile is activated and make it possible to deactivate it? The reason is to make the new programmer experience better, while core developers can configure the build with their Maven settings file.

Have a look at what @stefannegrea did in Metrics:
https://github.com/hawkular/hawkular-metrics/blob/master/rest-tests/pom.xml#L125

I'm not convinced deactivating the profile with a system setting (`-Drunning.service`) is necessary, it could be done with the profile arg actually (`-P !wildfly.deployment`). But details put aside, I believe it's a better solution.